src/App.scss

```diff
@@ -1,40 +1,11 @@
 @import '~bootstrap/scss/bootstrap';
```

Comment:
I know that this was added in another PR, but let's remove it since we're editing the file. We want to have this import in `index.scss` instead which is the top level SCSS file